src/actions/account.js

```diff
@@ -126,6 +139,36 @@ export const { initializeRecoveryMethod, validateSecurityCode, setupRecoveryMess
         wallet.validateSecurityCode.bind(wallet),
         () => defaultCodesFor('account.validateSecurityCode')
     ],
+    INIT_TWO_FACTOR: [
```

Comment:
seems like makes sense to have 2fa stuff in separate module

Reply:
I agree, however this would be a big refactor in it's current form given the reliance on all the wallet.js constants.

Probably need to scope out at least half a day + testing to ensure we don't miss any imports.

Reply:
TBD defer to @kcole16

Reply:
> would be a big refactor in it's current form given the reliance on all the wallet.js constants.

@mattlockyer not sure what are you referring to. Doesn't seem like importing constants from `wallet.js` would be a big deal.

Reply:
It's been done.